2/Mounts.rb

```diff
@@ -0,0 +1,11 @@
+# Сделать хеш, содеращий месяцы и количество дней в месяце. В цикле выводить те месяцы, у которых количество дней ровно 30
+
+require 'date'
+
+monts = Date::MONTHNAMES
+
+days_in_year = [31, 28, 31, 30, 31, 30, 31, 31, 30, 31, 30, 30]  
+
+year = monts.zip(days_in_year).to_h
+
+year.each {|key, _| puts key if year[key] == 30}
```

Comment:
```suggestion
year.each {|month, days_count| puts month if days_count == 30}
```
Лучше стараться называть временные переменные осмысленными именами (как и обычные переменные)

Reply:
Также при сериализации хэша, тебе доступны сразу и ключ и значение. Ты почему-то не стал использовать значение, а решил занового его находить через `year[key]`